content/en/community/_index.html

```diff
@@ -184,53 +184,53 @@ <h1 style="color:white !important">Upcoming Events</h1>
 </div>
 </div>
 
-<div class="meetups">
-<div class="meetupcol">
-<div class="meetuptext">
-<h1 style="text-align:left">Global Community</h1>
-With over 150 meetups in the world and growing, go find your local kube people. If one isn't near, take charge and create your own.
-</div>
-<a href="https://www.meetup.com/topics/kubernetes/">
-<div class="button">
-FIND A MEETUP
-</div>
-</a>
-<a name="news"></a>
-</div>
-</div>
-
-
-<!--
-<div class="contributor">
+<div class="meetups">
+<div class="meetupcol">
+<div class="meetuptext">
+<h1 style="text-align:left">Global Community</h1>
+With over 150 meetups in the world and growing, go find your local kube people. If one isn't near, take charge and create your own.
+</div>
+<a href="https://www.meetup.com/topics/kubernetes/">
+<div class="button">

```

Comment:
Please add alt text attribute
https://webaim.org/techniques/alttext/#functional

Reply:
> Please add alt text attribute
> https://webaim.org/techniques/alttext/#functional

@Rajakavitha1 Anchor tags are accessible by default and they don't support alt text because of this reason. I tried to check this with a screen reader, this should be fine.

Reply:
Great! then please ignore this.